controller/src/main/java/io/pravega/controller/server/eventProcessor/AutoScaleRequestHandler.java

```diff
@@ -57,7 +58,8 @@ public AutoScaleRequestHandler(final StreamMetadataTasks streamMetadataTasks,
         this.executor = executor;
     }
 
-    public CompletableFuture<Void> process(final AutoScaleEvent request) {
+    @Override
+    public CompletableFuture<Void> process(final AutoScaleEvent request, final EventProcessor.Writer<AutoScaleEvent> writer) {
```

Comment:
Same here, this writer doesn't seem to be used anywhere.

Reply:
replied above.. i am cleaning and refactoring this as part of 1738.. right now writer was needed by commit requesthandler so we have it here to keep the changes contained otherwise..